pkgs/by-name/ca/cargo-llvm-cov/package.nix

```diff
@@ -54,16 +41,21 @@ rustPlatform.buildRustPackage (finalAttrs: {
     inherit owner;
     repo = "cargo-llvm-cov";
     rev = "v${version}";
-    sha256 = "sha256-iJrnNDSMich5OzEbPgnQWLVz6Zj/MUIzEsaBzqVdoDg=";
+    sha256 = "sha256-0Dgcl1r+j86WVhkyJNJxUnk85vrtD1fBBxfJkV0mcAI=";
   };
 
   # Upstream doesn't include the lockfile so we need to add it back
-  postUnpack = ''
-    cp ${cargoLock} ${finalAttrs.src.name}/Cargo.lock
+  postPatch = ''
```

Comment:
Note: I changed this to `postPatch` because it seems to be the overwhelmingly common way to handle adding `Cargo.lock` in situations like this.